frontend/src/components/project/backlog.jsx

```diff
@@ -5,7 +5,7 @@ let Backlog = React.createClass({
 
   render() {
     let tasks = this.props.tasks.map((task) => {
-      return (<Task name={task.name} description={task.description} score={task.score} />);
+      return (<Task key={task.id} id={task.id} name={task.name} description={task.description} score={task.score} />);
```

Comment:
React throws a warning if there isn't a unique `key` property on repeated elements. This `key` property is not accessible through `this.props`, so we need `id` as well.